### PR TITLE
Screen/wishlist

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Fixed Scrolling Issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # zencart
+
 https://zencart.netlify.app
```
Comment (Megha-Joshi) [inline README.md:3]: The header is overriding the cards. Otherwise everything is fine.

**Review by Megha-Joshi — COMMENTED** [review]
The header of wishlist is overriding the cards.

**Review by srishtigautam21 — APPROVED** [review]
good to go

File: README.md (hunk 1 above)
Comment (sanjitsarkar) [reply README.md:3]: I did not get by overriding the cards

**Pull request merged**